### PR TITLE
Issue/3/sni callback

### PR DESCRIPTION
**Pull request by raintonr**

### CODE REVIEW

File: src/lib/webServer.ts [hunk 1]
```diff
@@ -133,7 +133,7 @@ export class WebServer {
                                 // This is very bad, and perhaps the adapter should also terminate itself?
                             }
                         }
-                        // TODO: How new certificates will be used?
+                        // contexts is now up to date and will be utilised in SNICallback - nothing more to do.
```
Comment (foxriver76) [inline src/lib/webServer.ts:136]: would just remove it entirely

**Review by foxriver76 — APPROVED** [review]
lgtm

File: src/lib/webServer.ts (hunk 1 above)
Comment (raintonr) [reply src/lib/webServer.ts:136]: ... and then have someone else come along and wonder how it works? Doesn't hurt to explain.

Comment (foxriver76) [reply src/lib/webServer.ts:136]: Ok can also leave it there maybe it’s just not my comment style to have them at the end of code, but fine for me here

**Pull request merged**